app/src/main/java/com/example/poste/activities/NewPostActivity.java

```diff
@@ -134,7 +134,9 @@ private void handleNewPost() {
             String folder_id = ((Folder) chooseFolderSpinner.getSelectedItem()).getId();
             if (folder_id.equals("0")) {
                 Log.d("NewPostActivity", "Default folder (0) selected");
-                Toast.makeText(this, "Select a folder to save post!", Toast.LENGTH_SHORT).show();
+                Toast.makeText(this,
+                        getString(R.string.pick_a_folder),
```

Comment:
This could be named a bit more descriptively, so reading through strings indicates its place of usage

Reply:
changed